temporal-sdk/src/main/java/io/temporal/client/WorkflowOptions.java

```diff
@@ -105,7 +105,7 @@ public static final class Builder {
 
     private List<ContextPropagator> contextPropagators;
 
-    private boolean disableEagerExecution;
+    private boolean disableEagerExecution = true;
```

Comment:
If we were to do it over again, I suspect we'd prefer `enableEagerExecution` here and I suspect that's what it'll be in all other SDKs. Is it worth breaking compatibility to make it more clearly opt-in? If we are willing to change it, I wonder if `enableEagerStart` is a better name to match Go.

Reply:
I agree if we were doing it from scratch I would change the name, but we aren't doing it from scratch and we need to keep in mind of compatibility guarantees so I wouldn't change the name since it's more trouble then it's worth

Reply:
:+1: I think at least for any other SDKs we implement this in, we can be more Go like and just let Java be the outlier